cmd/ako-main/main.go

```diff
@@ -307,6 +324,12 @@ func InitializeAKC() {
 	waitGroupMap["graph"] = wgGraph
 	wgStatus := &sync.WaitGroup{}
 	waitGroupMap["status"] = wgStatus
+
+	if lib.IsIstioEnabled() {
+		<-istioUpdateCh
+		c.IstioBootstrap()
```

Comment:
@aaha97: `istioWatcherEvents` will create secrets and AKO will catch that event in ingestion layer. Does explicit calling rest layer over here require?

Reply:
the Pki Profile and workload certificate need to be created before the queues are initialized